apps/lib/droplet/droplet-full.js

```diff
@@ -15170,21 +15177,30 @@ exports.Parser = Parser = (function() {
               currentlyCommented = false;
             }
           }
+          error = {
+            line: mark.location.line
+          };
           switch (mark.token.type) {
             case 'indentStart':
               if ((stack != null ? (ref7 = stack[stack.length - 1]) != null ? ref7.type : void 0 : void 0) !== 'block') {
-                throw new Error('Improper parser: indent must be inside block, but is inside ' + (stack != null ? (ref8 = stack[stack.length - 1]) != null ? ref8.type : void 0 : void 0));
+                error.type = "IncorrectBlockParent";
```

Comment:
For documentation and comments on this file, please look at this pr: https://github.com/droplet-editor/droplet/pull/214
(Comments get stripped from the minified js)